transports/quic/src/config.rs

```diff
@@ -71,19 +74,33 @@ pub struct Config {
 
     /// Parameters governing MTU discovery. See [`MtuDiscoveryConfig`] for details.
     mtu_discovery_config: Option<MtuDiscoveryConfig>,
+
+    webtransport_certhashes: Vec<Multihash<64>>,
 }
 
 #[expect(deprecated)]
 impl Config {
     /// Creates a new configuration object with default values.
-    pub fn new(keypair: &libp2p_identity::Keypair) -> Self {
+    pub fn new(keypair: &libp2p_identity::Keypair, webtransport_cert: Option<Certificate>) -> Self {
```

Comment:
Instead of breaking existing clients of this library, could this be changed to add two new functions, instead of changing the existing interface for people who, for whatever reason, don't want to use we transport yet? 

E.g. A `new_with_webtransport_cert` and a `with_webtransport_cert` builder function?

That would allow existing code to upgrade to a later version of this library, and add support only as they want.